### DISCUSSION TITLE
Is it possible to connect to streamdeck application?

### DISCUSSION THREAD

**Discussion by alchemistake** [StreamController/StreamController#301] [Ideas]
There is an app that makes your iPad to act like stream deck. is it possible to support it? If so how? can some one tldr it to me so i can check whether i can implement it or not?

**Comment by oparada1988** [comment]
Currently this is not supported

**Comment by alchemistake** [comment]
That was not my question.

**Reply by oparada1988** [reply]
its currently not in the timeline to add this as a feature. So its not supported at this moment. You can fork the project and implement it, then push it. You would need to check elgatos website to see if its even possible for a third party program to to connect to their own streamdeck app on your phone/ipad

**Comment by G4PLS** [comment]
This would be possible to do but it would require a few things:
- Better/more advanced StreamController api
- Intermediate host that communicates between StreamController and your phone
- An iPad/Andriod app that can communicate with **our** intermediate

**Reply by alchemistake** [reply]
TY for explaining what is needed im not gonna spend time on this.
